opensource/src/main/java/ua/gov/diia/opensource/helper/PinHelperImpl.kt

```diff
@@ -14,16 +14,13 @@ import javax.inject.Singleton
 @Singleton
 class PinHelperImpl @Inject constructor(
     private val biometric: Biometric,
-    private val settingsRepository: AppSettingsRepository,
+    private val settingsRepository: AppSettingsRepository

```

Comment:
Adding trailing comma is one of development technique. Maybe should to apply this changes by adding a rule of static linters like a detekt, ktlint